pkg/proxy/winkernel/proxier.go

```diff
@@ -780,15 +805,26 @@ func CleanupLeftovers() (encounteredError bool) {
 	return encounteredError
 }
 
-func (svcInfo *serviceInfo) cleanupAllPolicies(endpoints []proxy.Endpoint) {
+func (svcInfo *serviceInfo) cleanupAllPolicies(endpoints []proxy.Endpoint, isEndpointChange bool) {
```

Comment:
*nit:* Should we rename "isEndpointChange" to "isEndpointTerminated"?

Reply:
We wanted to inform here that the function is called as part of endpointmap change and not servicemap change